cockroachcloud/create-your-cluster.md

```diff
@@ -7,7 +7,9 @@ docs_area: deploy
 
 {% include cockroachcloud/filter-tabs/create-cluster-cloud.md %}
 
-This page walks you through the process of creating a {{ site.data.products.dedicated }} cluster. Note that only [{{ site.data.products.db }} Org Administrators](authorization.html#org-administrator-legacy) or users with Cluster Creator / Cluster Admin roles assigned at organization scope can create clusters. If you are a Developer and need to create a cluster, contact your {{ site.data.products.db }} Administrator.
+This page walks you through the process of creating a {{ site.data.products.dedicated }} cluster using Cloud Console. To use the Cloud API instead, refer to [Create a New Cluster](cloud-api.html#create-a-new-cluster).
```

Comment:
I've always had "the" before "Cloud Console". Do you have a preference for leaving it out?

Reply:
For some reason I really thought we had a style guide item about leaving the article off Cloud Console and DB Console, but I can't find any evidence of it. Sorry about that.